spec/Layers/FeatureLayer/FeatureLayerSpec.js

```diff
@@ -230,6 +258,8 @@ describe('L.esri.Layers.FeatureLayer', function () {
 
     layer.createLayers(multiPolygon);
 
+    expect(layer.getFeature(1).getLayers()[0].options.color).to.equal('black');
```

Comment:
i don't think we were having problems with setStyle/resetStyle before, but its a good test, so i followed your suggestion and laid it down in an additional commit